testing/cmdlineshell.test

```diff
@@ -95,3 +95,12 @@ do_execsql_test_on_specific_db testing/testing.db schema-2 {
 #        name TEXT,
 #        price REAL
 #    );"}
+
+# FIXME sqlite uses multicolumn output mode for display resulting in different spacing
+# do_execsql_test_on_specific_db testing/testing.db schema-1 {
```

Comment:
You can add this to `testing/shell.py`, is currently where all the `limbo` specific shell output tests are. But maybe still leave this here with the FIXME for future reference to match `sqlite3` behavior.

Reply:
Done, thanks for clarifying